### ISSUE TITLE
import cycle detected?

### ISSUE THREAD

**Issue by JalonSolov** [vlang/v#4442]
Building V this morning shows something I haven't seen before:

```
V self compiling...
import cycle detected between the following modules:

 * v.table -> v.ast
 * v.table -> v.builder
 * v.builder -> v.table
 * v.ast -> v.table
make modules
```

**Comment by SleepyRoy** [comment]
I didn't test on my computer. It may come from 8a1248b by @spytheman.

**Comment by JalonSolov** [comment]
Just confirming this is halfway fixed by commit 3cc70094401571d209a89669e02e4a49e2ef73a7

**Comment by spytheman** [comment]
Solved in https://github.com/vlang/v/commit/7e4e3abc2ca38d2e784ad3a29f8cca972ec1c3df
